tutorials/clinical/30_ecog.py

```diff
@@ -115,9 +115,11 @@
 # (along with xy positions of each electrode in the image), so that later
 # we can plot frequency band power on top of it.
 
+sensor_colors = np.random.random([len(raw.info["ch_names"]), 3])
```

Comment:
adding random colors is not really appropriate for a tutorial (which should generally show best practices, or at least sensible choices that most users would likely want to make).  Can you instead demonstrate highlighting one or a few sensors in one or a few colors in order to make a particular point about the data?